test/runtests.jl

```diff
@@ -1,4 +1,11 @@
 using Test
 using HierarchicalRouting
 
+include("setup.jl")
+
+const (TEST_PROBLEM, TEST_SOLUTION) = initialize_instances()
```

Comment:
This only works once if running tests directly - in all following cases it will raise an error because constants cannot be redefined in the same session.